app/models/mixins/authentication_mixin.rb

```diff
@@ -79,14 +79,7 @@ def authentication_service_account(type = nil)
   end
 
   def required_credential_fields(type)
-    case type.to_s
-    when "bearer"
-      [:auth_key]
-    when "hawkular"
-      []
-    else
-      [:userid]
-    end
+    []
```

Comment:
@chessbyte @abonas 
> So, do we need a similar override for hawkular or the default [:userid] will work for it?

Sorry, I didn't understand the question.
The _hawkular middleware_ use the `:default `authentication type, it requires the `[:unierid, :password]` fields. it will work with the new default `[:userid]`.